src/test/java/com/yegor256/tojos/MnCsvTest.java

```diff
@@ -122,4 +129,79 @@ void putsKeyFirst(@Mktmp final Path temp) throws IOException {
             )
         );
     }
+
+    @RepeatedTest(5)
+    void handlesConcurrentModificationInDupMethod(@Mktmp final Path temp) {
+        final Collection<Map<String, String>> rows = new ArrayList<>(10_000);
+        final AtomicReference<Throwable> exc = new AtomicReference<>();
+        final Path path = temp.resolve("key-test.json");
+        final Mono csv = new MnCsv(path);
+        final int threads = 3;
+        new Together<>(
+            threads, thead -> {
+            switch (thead) {
+                case 0:
+                    add(rows);
+                    break;
+                case 1:
+                    dup(csv, rows, exc);
+                    break;
+                case 2:
+                    try {
+                        rem(rows);
+                    } catch (final ConcurrentModificationException ex) {
```

Comment:
@VladMilash what's the point of catching and then ignoring? If you remove this `catch`, the test will pass.